src/resolvers/Mutation/sendMembershipRequest.ts

```diff
@@ -78,8 +78,15 @@
 
     // Checks if the user is blocked
     const user = await User.findById(context.userId).lean();
+    if (user === null) {
+      throw new errors.NotFoundError(
+        requestContext.translate(USER_NOT_FOUND_ERROR.MESSAGE),
+        USER_NOT_FOUND_ERROR.CODE,
+        USER_NOT_FOUND_ERROR.PARAM,
+      );
+    }
```

Comment:
If the following check is not included, the code throws an error when evaluating the user, indicating it might be null:

```javascript 
    if (
      organization.blockedUsers.some((blockedUser) =>
        new mongoose.Types.ObjectId(blockedUser.toString()).equals(user._id),
      )
    ) {
      throw new errors.UnauthorizedError(
        requestContext.translate(USER_NOT_AUTHORIZED_ERROR.MESSAGE),
        USER_NOT_AUTHORIZED_ERROR.CODE,
        USER_NOT_AUTHORIZED_ERROR.PARAM,
      );
    }
```
Adding this check prevents the error.